docs/docsite/rst/userguide/teams.rst

```diff
@@ -97,6 +104,7 @@ Selecting the **Roles** view displays a list of the permissions that are current
 |Teams - permissions list|
 
 .. |Teams - permissions list| image:: ../common/images/teams-permissions-sample-roles.png
+   :alt: Permissions List
 
```

Comment:
Missing image after Step 1: teams-users-add-permissions-form.png. 
I suggest something like:
"Add Teams Permissions Wizard step 1, choose the resource type."